contrib/hadoop-store-builder/src/java/voldemort/store/readonly/mr/azkaban/VoldemortSwapJob.java

```diff
@@ -121,15 +121,15 @@ public void run() throws Exception {
 
         // While processing an admin request, HDFSFailedLock could take long time because of multiple HDFS operations,
         // especially when the name node is in a different data center. So extend timeout to 5 minutes.
-        AdminClientConfig adminConfig = new AdminClientConfig().setMaxConnectionsPerNode(cluster.getNumberOfNodes())
+        AdminClientConfig adminConfig = new AdminClientConfig().setMaxConnectionsPerNode(cluster.getNumberOfNodes()) // TODO: Figure out why do we need this config?? -FGV
```

Comment:
The number of nodes is a bug. but since Voldemort grows connection on a need basis, I never bothered to fix it. It should be 2 or 3 and never more than that. But it does not hurt as there should not be parallel connections using it.